GamebookEngine/Views/Editing/Page List/PagesTableViewController.xib

```diff
@@ -21,12 +22,17 @@
             <simulatedNavigationBarMetrics key="simulatedTopBarMetrics" prompted="NO"/>
             <color key="separatorColor" systemColor="separatorColor" red="0.23529411759999999" green="0.23529411759999999" blue="0.26274509800000001" alpha="0.28999999999999998" colorSpace="custom" customColorSpace="sRGB"/>
             <inset key="separatorInset" minX="20" minY="0.0" maxX="20" maxY="0.0"/>
+            <searchBar key="tableHeaderView" contentMode="redraw" placeholder="Search" translucent="NO" id="Vdq-kV-sqO">
+                <rect key="frame" x="0.0" y="0.0" width="414" height="44"/>
+                <autoresizingMask key="autoresizingMask" widthSizable="YES" flexibleMaxY="YES"/>
+                <textInputTraits key="textInputTraits"/>
+            </searchBar>
             <point key="canvasLocation" x="327" y="4"/>
         </tableView>
     </objects>
     <resources>
         <namedColor name="background">
-            <color red="0.10999999940395355" green="0.10999999940395355" blue="0.11400000005960464" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+            <color red="0.94117647058823528" green="0.93725490196078431" blue="0.96470588235294119" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```

Comment:
What's this?

Reply:
Not sure why this was changed. You can disregard it.